kaldb/src/main/java/com/slack/kaldb/metadata/core/KaldbMetadataStore.java

```diff
@@ -59,7 +59,7 @@ public KaldbMetadataStore(
     // Create the path to the store in ZK. However, since 2 different processes may create
     // a path at the same time, ignore the exception if node already exists.
     try {
-      metadataStore.create(storeFolder, "", true).get();
+      metadataStore.create(storeFolder, "{}", true).get();
```

Comment:
I think the right fix is to force a default value for the root nodes.

Reply:
In our `ZookeeperMetadataStoreImpl.createImpl` [we attempt to call](https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/metadata/zookeeper/ZookeeperMetadataStoreImpl.java#L242) `curator.create()....forPath(path, data.getBytes())`:

The curator framework provides two methods when creating a node - ` forPath(String path, byte[] data)` and ` forPath(String path)`. Per the javadoc on `PathAndBytesable` we should be using the latter if there is no data, which will use the configured `Builder.defaultData` (usually `byte[0]`).

In the `createImpl` we should just check to see if there's any data that is attempting to be set, and if it use the `forPath(String path)` call instead.